server/endpoint/error.go

```diff
@@ -0,0 +1,12 @@
+package endpoint
+
+import (
+	"github.com/juju/errgo"
+)
+
+var invalidConfigError = errgo.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return errgo.Cause(err) == invalidConfigError
+}
```

Comment:
Uncertain. I would like to move the version endpoint and service to microkit. This is a lot of boilerplate we can safe.